wthik.py

```diff
@@ -127,17 +124,22 @@ def travel_schedule(service, calendar_id, num_events=5):
 
 def help_response():
     resp = MessagingResponse()
-    resp.message("""Ask me "Where is Kelley?" to see my current whereabouts or "Travel schedule" to see what's coming up.""")
+    resp.message("""Ask me "Where is Matt?" to see my current whereabouts or "Travel schedule" to see what's coming up.""")
```

Comment:
Let's update this to `TRAVELER` while we're at it.

Reply:
Oh yeah, I missed that one.